Example/Backpack/View Controllers/BPKButtonsViewController.m

```diff
@@ -20,9 +20,12 @@
 
 #import <Backpack/Icon.h>
 #import <Backpack/Color.h>
+#import <Backpack/Label.h>
 
 NS_ASSUME_NONNULL_BEGIN
 @interface BPKButtonsViewController ()
+@property (strong, nonatomic) IBOutletCollection(UIView) NSArray *contentViews;
```

Comment:
I appreciate "contentViews" is a bit of a confusing term as the norm is to have a single `contentView`. Maybe there's a better name?